content/sensu-go/5.13/release-notes.md

```diff
@@ -38,6 +39,16 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.13.1 release notes
+
+**September 10, 2019** &mdash; The latest release of Sensu Go, version 5.13.1, is now available for download.
+This release is a stability release with bug fixes for multi-build asset definitions causing a panic when no matching filters are found.
+See the [upgrade guide][1] to upgrade Sensu to version 5.11.1.
+
+**FIXES:**
+
+- Multi-build asset definitions with no matching filters will no longer cause a panic.
```

Comment:
There is also a fix for OIDC not working on 5.13.0, see:
https://github.com/sensu/sensu-enterprise-go/commit/0dd79f311f85ab9a2b337b4ccea9ed4671357a7b#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR8-R11